asciinema/config.py

```diff
@@ -33,7 +33,11 @@ def api_token(self):
             try:
                 return self.config.get('user', 'token')
             except (configparser.NoOptionError, configparser.NoSectionError):
-                raise ConfigError('no API token found in config file')
+                environment_variable_fallback = self.env.get('ASCIINEMA_TOKEN')
```

Comment:
Can we call it `ASCIINEMA_API_TOKEN`, to be consistent with `api.token` from the config file, and with parallel `api.url, ASCIINEMA_API_URL`?